components/camel-log/src/main/java/org/apache/camel/component/log/LogEndpoint.java

```diff
@@ -118,6 +118,8 @@
     @UriParam(label = "formatting", enums = "Default,Tab,Fixed", defaultValue = "Default",
               description = "Sets the outputs style to use.")
     private DefaultExchangeFormatter.OutputStyle style = DefaultExchangeFormatter.OutputStyle.Default;
+    @UriParam(defaultValue = "false", description = "Log exchange body as-is (ex. log(\"${body}\").")
```

Comment:
Lets use the same description:
```
If enabled only the body will be printed out
```

To make it stand out that all the other options are not considered then